extendeddatacrossword.go

```diff
@@ -6,32 +6,30 @@ import (
 	"fmt"
 )
 
+// axis represents which of a row or col.
```

Comment:
Capitalize.

Reply:
Fixed in [bd5044c](https://github.com/celestiaorg/rsmt2d/pull/80/commits/bd5044c996b04e39353775856cd8f5f6a158a182)